R/results_habitat_association.R

```diff
@@ -150,6 +150,13 @@ results_habitat_association <- function(pattern, raster,
 
     }
 
+    # warning if NA are present
+    if (anyNA(raster@data@values)) {
```

Comment:
This adds a dependency to R >= 3.2.0

Reply:
Oh really? I didn't know that. Fixed in Description (3f9c5bb).

Reply:
Oh my bad, you're right. After looking into the release notes, `anyNA()` was added in R 3.1.0. R 3.2.0 says:

> anyNA() gains a recursive argument.

The release notes are here: https://cran.r-project.org/doc/manuals/NEWS.3

Reply:
It requires R>=3.1.0 though

Reply:
Yes, it does